runtime-decompiler/src/main/java/org/jrd/frontend/MainFrame/BytecodeDecompilerView.java

```diff
@@ -138,6 +137,15 @@ protected Void doInBackground() throws Exception {
         );
 
         topComboBox = new JComboBox<DecompilerWrapperInformation>();
+        topComboBox.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent actionEvent) {
+                if(filteredClassesJlist.getSelectedIndex() != -1){
+                    ActionEvent event = new ActionEvent(this, 1, filteredClassesJlist.getSelectedValue());
+                    bytesActionListener.actionPerformed(event);
+                }
```

Comment:
else { clean window} ?

Reply:
I don't think that punishing the user by closing the BytecodeDecompilerView for changing decompilers is the right thing to do. Or...what do you mean by cleaning the window?